pkg/project/project_loader.go

```diff
@@ -32,7 +33,9 @@ import (
 // it also resolves all project dependencies
 // if no -p parameter specified, then it creates a list of all projects
 func LoadProjectsToDeploy(specificProjectToDeploy string, apis map[string]api.Api, path string, fileReader util.FileReader) (projectsToDeploy []Project, err error) {
-	projectsFolder := filepath.Join(".", path)
+	//projectsFolder := filepath.Join(".", path)
+	projectsFolder := path
+	//absolute path solution, test if the folder exist leave it as it is, otherwise add the dot
```

Comment:
This sounds like this is left over from some local test? 
Should this change go in or be dropped? 

If it should be merged, please make sure it's a dedicated commit, as it seems unrelated to the download functionality(?)

Reply:
Hi Nico Just implemented your suggestions alongside the integration with the new api 2.0 in a new brach with a new pull request since there were quite a few changes in main.go